src/components/visualizations/chart/legend/LegendItem.tsx

```diff
@@ -18,7 +18,7 @@ export default class LegendItem extends React.Component<any, any> {
         const enableBorderRadius = isLineChart(itemChartType) || isAreaChart(itemChartType);
 
         const iconStyle = {
-            borderRadius: enableBorderRadius ? "50%" : "none",
+            borderRadius: enableBorderRadius ? "50%" : "0",
```

Comment:
That's a workaround solution. There is an issue when switching chart type from "line"/"area" -> "column", the border radius of legend icon is not updated (from "50%" -> "none")